config/nuttx/stm32f4dis/jerry-memstat.diff

```diff
@@ -0,0 +1,55 @@
+diff --git a/jerry-core/api/jerry.c b/jerry-core/api/jerry.c
+index 1032881..88db6a3 100644
+--- a/jerry-core/api/jerry.c
++++ b/jerry-core/api/jerry.c
+@@ -149,7 +149,7 @@ jerry_init (jerry_init_flag_t flags) /**< combination of Jerry flags */
+   }
+
+   /* Zero out all members. */
+-  memset (&JERRY_CONTEXT (JERRY_CONTEXT_FIRST_MEMBER), 0, sizeof (jerry_context_t));
++  // memset (&JERRY_CONTEXT (JERRY_CONTEXT_FIRST_MEMBER), 0, sizeof (jerry_context_t));
```

Comment:
Why this is commented out?

Reply:
It need to prevent the overflow of the counter. When `jerry_init` called, the allocated memory is not zero, because the IoT.js did some allocation for the environment. So we should not zero out memory statistics here, because it will cause an overflow after the `free` function calls.